### DISCUSSION TITLE
Repeat Options: tabs option instead of just repeated rows only

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#70966] [Dashboards, Panels & Visualizations]
<img width="186" alt="image" src="https://github.com/grafana/grafana/assets/900951/bf2d0995-0f60-4267-a116-8569c7efe3db">

Greetings,

It would be awesome if the repeat options can be mapped to tabs as another option to endless rows. There are cases where one has a busy dashboard and repeating panels would make it pretty unusable. So was thinking a tabular result would be much cleaner?

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82593. Feel free to continue the discussion around this there. Thank you!
